### PR TITLE
Upgrade to Decidim 0.27

### PR DESCRIPTION
**Pull request by NicoleeSo13**
This PR upgrades the Decidim Gava App to Decidim v0.27.4.

Release: https://github.com/decidim/decidim/releases :
- 0.27.4 : https://github.com/decidim/decidim/releases/tag/v0.27.4
- 0.27.3 : https://github.com/decidim/decidim/releases/tag/v0.27.3
- 0.27.2 : https://github.com/decidim/decidim/releases/tag/v0.27.2
- 0.27.1 : https://github.com/decidim/decidim/releases/tag/v0.27.1
- 0.27.0: https://github.com/decidim/decidim/releases/tag/v0.27.0 

Note that many Decidim configuration has moved from the `config/initializers/decidim.rb` to ENV variables. See https://github.com/decidim/decidim/pull/8725 for a full list of ENV vars.

To execute after deploy:
https://github.com/decidim/decidim/blob/release/0.27-stable/CHANGELOG.md#3-one-time-actions

### CODE REVIEW

File: config/deploy/production.rb
Comment (laurajaime) [inline config/deploy/production.rb:1]: we can remove deploy folder.

File: config/environments/staging.rb
Comment (laurajaime) [inline config/environments/staging.rb:1]: we can remove this file

File: config/locales/es.yml [hunk 1]
```diff
@@ -62,3 +62,6 @@ es:
   errors:
     messages:
       not_in_district: El código postal no pertenece al distrito
+  census_authorization_handler:
+    not_old_enough: Menor de 16 años
+    not_in_census: No empadronado
```
Comment (laurajaime) [inline config/locales/es.yml:67]: Why you add this translations?

File: db/schema.rb [hunk 2]
```diff
@@ -535,6 +546,56 @@
...
+  end
+
+  create_table "decidim_dummy_resources_nested_dummy_resources", force: :cascade do |t|
+    t.jsonb "translatable_text"
+    t.string "title"
+    t.bigint "dummy_resource_id"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+  end
+
```
Comment (laurajaime) [inline db/schema.rb]: what are this tables?

**Review by laurajaime — COMMENTED** [review]
Good job. Only some comments

File: config/locales/es.yml (hunk 1 above)
Comment (NicoleeSo13) [reply config/locales/es.yml:67]: To fix tests that were failing in 0.26 and that were failing after update

File: db/schema.rb (hunk 2 above)
Comment (NicoleeSo13) [reply db/schema.rb]: Scheme changes was created during rails db:migrate. Won't work without them.

File: config/deploy/production.rb
Comment (NicoleeSo13) [reply config/deploy/production.rb:1]: Removed

File: config/environments/staging.rb
Comment (NicoleeSo13) [reply config/environments/staging.rb:1]: Removed

File: db/schema.rb (hunk 2 above)
Comment (laurajaime) [reply db/schema.rb]: be careful with mentions!

File: config/locales/es.yml (hunk 1 above)
Comment (laurajaime) [reply config/locales/es.yml:67]: What tests?

File: db/schema.rb (hunk 2 above)
Comment (NicoleeSo13) [reply db/schema.rb]: OK, I have deleted this three additional columns, they do not seem to exist in other updates (and they don't have proper migrations for them). They were probably added during dummy app tests.

File: config/locales/es.yml (hunk 1 above)
Comment (NicoleeSo13) [reply config/locales/es.yml:67]: There were a few tests in 0.26 giving missing translation error. Upgrading to 0.27 haven't fixed them. Adding this translations fixed them.

**Review by laurajaime — APPROVED** [review]

**Pull request merged**